pkg/ui/src/views/shared/components/sortedtable/index.tsx

```diff
@@ -121,15 +131,19 @@ export class SortedTable<T> extends React.Component<SortedTableProps<T>, {}> {
   );
 
   render() {
-    const { data, sortSetting, onChangeSortSetting } = this.props;
+    const { data, sortSetting, onChangeSortSetting, loading } = this.props;
+    if (!_.isNil(loading) && loading) {
```

Comment:
this can just be `if (loading)`, `!_.isNil()` is guaranteed true if loading is true.

But also, see above, this should be a separate component which wraps the table.